README.md

```diff
@@ -1,18 +1,55 @@
 # eshop-api
+
 unified api to common eshops
 
 ## Development Setup
 
-* Install PostgreSQL via Docker to IP 192.168.99.100 and port 32769
-* Connect to your database
+Running locally isn't currently supported. We deploy a test version of the app to [now](https://zeit.co/) and then test that in slack. There is an Airtable base for testing and also a test Slack App.
```

Comment:
Why isn't it supported?

Reply:
Because it hasn't been setup that way. Slack app can't communicate with localhost, but ofc there are ways to make it work. Should I commit some time to this? I wasn't sure if it is worth it since it is a small internal project and it hasn't been setup so far.

Reply:
Interesting. I always used to run this locally via ngrok.

Reply:
Yes, I ended up also using ngrok yesterday morning. I thought it would be more complicated, but it turned out to be pretty straightforward. I will update the README today.